ui/css/pods-flexible-relationships.css

```diff
@@ -0,0 +1,24 @@
+#wpadminbar, #adminmenuback, #adminmenuwrap, #preview-action, #wpfooter {
+	display: none;
+}
+
+html.wp-toolbar {
+	padding-top: 0 ;
+}
+
+#wpbody-content {
+	padding-bottom: 0;
+}
+
+.auto-fold #wpcontent, .auto-fold #wpfooter {
+	margin-left: 0;
+}
+
+#wpcontent {
+	padding: 0;
+	margin: 0 1.25em;
+}
+
+.wrap {
+	margin: 0;
+}
```

Comment:
@nicdford @pglewis I'd also like to see `body.edit-tags-php #col-right { display: none; }` to hide the terms list and `body.edit-tags-php #col-left { width: 100%; }` to make the form full modal width.